akka-http-tests/src/test/scala/akka/http/scaladsl/coding/CoderSpec.scala

```diff
@@ -113,20 +112,22 @@ abstract class CoderSpec extends WordSpec with CodecSpecSupport with Inspectors
     }
 
     "shouldn't produce huge ByteStrings for some input" in {
-      val array = new Array[Byte](10) // FIXME
-      util.Arrays.fill(array, 1.toByte)
+      val array = new Array[Byte](100007)
+      val random = ThreadLocalRandom.current()
```

Comment:
Why random data? I guess it doesn't really matter but the original test tried (eh, very hard) to emulate a zip bomb which would use well-compressible data.

Reply:
I chose random data, as I wanted to detect frames being resend, etc, so each frame needed to be unique. I didn't fancy creating an array with 100007 bytes of data manually.

Reply:
I see, thanks for the explanation and the fix!